### DISCUSSION TITLE
Allgemeine Frage

### DISCUSSION THREAD

**Discussion by Trunks1982** [camueller/SmartApplianceEnabler#346] [Allgemein]
Hi Axel,
ich hätte mal ne Frage wie sieht der semp Abschaltbefehl aus der vom SHM kommt?

**Comment by camueller** [comment]
`DeviceId` und Hostname müssen angepasst werden:
```
curl -X POST -d '<EM2Device xmlns="http://www.sma.de/communication/schema/SEMP/v1"><DeviceControl><DeviceId>F-00000001-000000000001-00</DeviceId><On>false</On></DeviceControl></EM2Device>' --header 'Content-Type: application/xml' http://raspi:8080/semp
```

**Comment by Trunks1982** [comment]
Ich hab noch ne Frage, wenn Geräte Typ Wärmepumpe ausgewählt ist, wäre das von SMA so das Sie nur einmal angeschaltet wird und bis Maxrunntime ignoriert und Sma sie gar nicht abschaltet?
Also das sie nur nach Startzeit und Endzeit geht?

**Comment by camueller** [comment]
Warum sollte sie nur einmal eingeschaltet werden? Solange die das Zeitfenster passt und maxRunningTime>0 ist schickt der SHM einen Einschaltbefehl.

**Reply by Trunks1982** [reply]
Das ist ist richtig.Meine eher das abschalten.Hatte im PV Forum was gelesen wo du mit dem SAE angefangen hast.Ist nur ne nur generelle Frage, ob der bei Auswahl Wärmepumpe der SHM den MaxRunningTime nicht beachtet sondern nur nach earlieststart und latesend geht.
